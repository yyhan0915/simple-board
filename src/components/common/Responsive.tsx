import React from 'react';
import styled from 'styled-components';
import { device } from '../../styles/palette';

interface ResponsiveProps {
    children?: unknown;
}

const ResponsiveBlock = styled.div`
    margin: 0 auto;
    position: relative;
    min-height: 100vh;
    width: 100%;

    @media ${device.mobile} {
    }

    @media ${device.laptop} {
    }
`;

const Responsive: React.FC<ResponsiveProps> = ({ children, ...rest }) => {
    return <ResponsiveBlock {...rest}>{children}</ResponsiveBlock>;
};

export default Responsive;
