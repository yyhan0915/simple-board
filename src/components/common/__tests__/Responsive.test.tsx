import { shallow } from 'enzyme';
import Responsive from '../Responsive';
import React from 'react';

describe('Responsive changes the text after click', () => {
    it('rendered without Crash', () => {
        const wrapper = shallow(<Responsive />);
        expect(wrapper).toMatchSnapshot();
    });
});
