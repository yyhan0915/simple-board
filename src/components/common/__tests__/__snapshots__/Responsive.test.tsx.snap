// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Responsive changes the text after click rendered without Crash 1`] = `ShallowWrapper {}`;
